src/main/java/com/streisky/virtualshowcasebackend/exception/VirtualShowcaseNotFoundException.java

```diff
@@ -1,6 +1,6 @@
 package com.streisky.virtualshowcasebackend.exception;
 
-public class VirtualShowcaseNotFoundException extends Exception {
+public class VirtualShowcaseNotFoundException extends RuntimeException {
```

Comment:
I am quite confused here. Since these Exceptions are caught on the `RestControllerAdvice` are they considered checked or unchecked? ([context](https://stackoverflow.com/questions/19857008/extending-exception-runtimeexception-in-java))

Reply:
As far as I know, in this version of spring boot that I'm using (3.1.4), there is no difference, and it is possible to handle the Exception class in RestControllerAdvice.

I decided to keep RuntimeException because all the examples I saw extends this class and not Exception.